eventually/authentication/apps.py

```diff
@@ -1,5 +1,13 @@
+"""
+Application configuration
```

Comment:
this module's docstring is not necessary because of apps.py is been ignored by pylintrc's field.

Reply:
Fixed